src/observable.js

```diff
@@ -114,6 +114,52 @@ extend(Observable.prototype, {
     return this._off(ANY, fn);
   },
 
+  subscribe(observer) {
+    const _this = this;
+    let closed = false;
+
+    if (typeof observer === 'function') {
+      observer = {
+        next: observer,
+        error: arguments[1],
+        complete: arguments[2]
+      };
+    }
+
+    if (observer.next) {
+      this.onValue(observer.next);
+    }
+
+    if (observer.error) {
+      this.onError(observer.error);
+    }
+
+    if (observer.end) {
+      this.onEnd(observer.complete);
+    }
+
+    return {
```

Comment:
That's strange. In https://github.com/rpominov/kefir/blob/master/src/interop/to-es-observable.js we return function from `.subscribe()` not an object. Either here or there we have an incompatibility with the spec.